src/components/Message/MessageText.js

```diff
@@ -40,6 +40,16 @@ const MessageTextComponent = (props) => {
   const hasReactions = messageHasReactions(message);
   const hasAttachment = messageHasAttachments(message);
   const handleReaction = useReactionHandler(message);
+  const messageText = useMemo(() => {
+    if (!message?.text) {
+      return null;
+    }
```

Comment:
the null check exists on the renderText fn

Reply:
👍  Improved here: https://github.com/GetStream/stream-chat-react/pull/452/commits/4cc8993e3804b8fee46d3cb95fd6816a518ae044